garageScript/c0d3-app
# Remove username from AppNav

Closes #2096 

Merging this PR will remove the username from the `AppNav` because it adds no value as the username also exist in the dropdown menu:

<img width="258" alt="image" src="https://user-images.githubusercontent.com/35906419/189174938-59d9684f-b92d-44d8-90af-13f78d4440cd.png">

**To test it:**
1. Go to `/curriculum` page
2. Observe the dropdown menu at the top-right corner
